### DISCUSSION TITLE
Categories should support multiple URIs for different contexts

### DISCUSSION THREAD

**Discussion by Mosnar** [craftcms/cms#11320] [Ideas] [answered]
Categories can be applied to a wide variety of content types - a single URI for a category implies one category group can only ever apply to one content type. 

Consider a website with sections for "Blog", "News", and "Podcasts" - these may make sense to share a common category group but have different URLs such as:
```
site.com/blog/cryptocurrency
site.com/news/cryptocurrency
site.com/podcast/cryptocurrent
```

Currently, I would need to either create & maintain 3 disjointed category groups or come up with a questionable custom-routing solution.

To be honest, I'm not sure what a good solution looks like for this, but I think it's worth thinking about.

**Comment by brandonkelly** [comment, marked as answer]
`category.url` should only give you a single URL though right? Which is where the URI comes in.

You can still pull the category in from additional places, including other pages that appear to be dedicated to the category. You would just need to set up the route yourself, and query the category from the template.

**Reply by Mosnar** [reply]
That's a good point... my idea is half-baked. I'll check back in next year when I've had more time think about it.
